css/index.css

```diff
@@ -143,4 +143,155 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+
+header{
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    margin: 10px;
+    padding: 10px;
+    align-items: center
+}
+
+.nav-container {
+    display: flex;
+    justify-content: space-around;
+}
+
+.logo{
+ display: flex;
+ align-items: center;
+ justify-content: center
+}
+
+nav {
+    display: flex;
+    justify-content: space-evenly;
+    align-items: flex-end;
+    margin-top: 20px;
+}
+
+nav a{
+    padding: 0px 40px;
+    text-decoration: none;
+    color: lightslategrey
+}
+
+.banner-container {
+    display: flex;
+    justify-content: flex-start;
+    margin: 30px 0px
+}
+
+.middle-content .boxes .box-1{
+    background-color: #008080
+}
+.middle-content .boxes .box-2{
+    background-color: gold
+}
+.middle-content .boxes .box-3{
+    background-color:cadetblue;
+}
+.middle-content .boxes .box-4{
+    background-color: coral;
+}
+.middle-content .boxes .box-5{
+    background-color: crimson;
+}
+.middle-content .boxes .box-6{
+    background-color: forestgreen;
+}
+.middle-content .boxes .box-7{
+    background-color: darkorchid;
+}
+.middle-content .boxes .box-8{
+    background-color: hotpink
+}
+.middle-content .boxes .box-9{
+    background-color: indigo;
+}
+.middle-content .boxes .box-10{ 
+    background-color: dodgerblue;
+}
+
+section{
+    margin: 20px 0
+}
+.container-1 {
+    margin: 15px 0;
+    border-bottom: 1px dashed black
+}
+p{
+    margin: 20px 0
+}
+.container-2 {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    width: 100%;
+    align-items: center
+}
+
+.wrapper-left,.wrapper-right{
```

Comment:
Why do you need to have 2 different parallel classes that are styled exactly the same? Why don't use just one of it?